### PR TITLE
Bans List Layout Frame

### PR DESCRIPTION
**Pull request by mordent-goonstation**
## About the PR 

I spent a not-okay amount of time getting this to layout in a way that was sane with the scroll how I want. IE11 is a riot.

Of interest: "Cell" component (will probably need renaming) that is the start of something akin to a config-driven grid. This could have application for similar interfaces, such as the Player Panel, as the point of it is to be feature-rich without requiring people to implement everything.

To be clear, this is very much a WIP. Everything is still "in flight" in that it might change, but this builds so...

## Why's this needed? 

*Plenty* more to do, but this is an incremental step that will let someone with a configured v2 API connection start seeing some stuff.

### CODE REVIEW

File: tgui/packages/tgui/interfaces/BanPanel/BanList/Cell.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { InfernoNode } from 'inferno';
```
Comment (ZeWaka) [inline tgui/packages/tgui/interfaces/BanPanel/BanList/Cell.tsx:1]: smh smh why isn't this in common smh

**Review by ZeWaka — APPROVED** [review]

**Review by Tarmunora — APPROVED** [review]
no issues with light-mode readability

**Comment by ZeWaka** [comment]
image
current

**Comment by ZeWaka** [comment]
😀

**Pull request closed without merging**